src/Altinn.AccessManagement.Core/Models/DelegationChange.cs

```diff
@@ -47,7 +47,18 @@ public class DelegationChange
         /// Gets or sets the user id of the user that performed the delegation change (either added or removed rules to the policy, or deleted it entirely).
         /// </summary>
         [JsonPropertyName("performedbyuserid")]
-        public int PerformedByUserId { get; set; }
+        public int? PerformedByUserId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the party id of the user that performed the delegation change (either added or removed rules to the policy, or deleted it entirely).
+        /// </summary>
+        [JsonPropertyName("performedbypartyid")]
+        public int? PerformedByPartyId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the timestamp for when the change was performed
+        /// </summary>
+        public DateTime PerformedDateTime { get; set; }
```

Comment:
So this is just at input property but never set when reading from the DelegationChange database, as it will then be the Created property? 

Any reason to not just use the existing Created property then?

Reply:
Good catch I did not see it had a Created as it was also used for reading from db will change

Reply:
I also notice that Created is declared DateTime so no null value is possible so the insert methods has traditionally not set this but the get methods has fetched it now one insert sets it but it sets it always if not provided it is set to Now.